src/stacks-hierarchy/Root_PurchaseOverviewScreen/use-offer-state.ts

```diff
@@ -233,8 +233,13 @@ export function useOfferState(
               },
             });
           }
-        } catch (err) {
-          const errorType = getAxiosErrorType(err);
+        } catch (err: any) {
+          const isNotAvailableError =
+            err.response?.status === 400 &&
+            err.response?.data === 'NotAvailable';
+          const errorType = isNotAvailableError
+            ? 'not-available'
+            : getAxiosErrorType(err);
```

Comment:
Do we expect the NotAvailable error to be unique to offers?
If not, it could be an idea to handle this inside `getAxiosErrorType`.

Since `'empty-offers'` isn't being used anymore, that would also make it possible to remove the `OfferError` type and just use `ErrorType` directly.

Reply:
> Do we expect the NotAvailable error to be unique to offers?
If not, it could be an idea to handle this inside getAxiosErrorType.

The 'NotAvailable' is unique to offers endpoint in the ticket legacy service. Platform team will look at a more proper fix when refactoring to the planned sales services.

> Since 'empty-offers' isn't being used anymore, that would also make it possible to remove the OfferError type and just use ErrorType directly.

It could be possible, but it would need the empty-offers error to be "masked" as an axios error, which isn't preferable.